starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <title>Apple Pie Recipe</title>
+    <link
+      rel="stylesheet"
+      href="/starter-code/apple-pie-starter-code/css/main.css"
+    />
+  </head>
+  <body class="body-align">
```

Comment:
the entire page in the example has a `padding` from both left and right.

Reply:
Also in the paragraphs try to add `padding` from both left and right, or change the `width` of your div to make it look more similar to the example.